common-libs/common/src/shared/seedTools.ts

```diff
@@ -130,6 +128,18 @@ export const convertDecryptedSeedBufferToString = (decryptedSeed: Buffer): strin
   return `${seedHex}++jolo`
 }
 
+export const processAnchoredElemDidSeed = (parsedSeed: ParseDecryptedSeedResult, did: string) => {
```

Comment:
The name of this method is misleading, something like `extendSeedWithDid` would be more appropriate.